Visualizations-TablePanel: added YouTube link to Table Panel

srperf opened the pull request:
**What is this feature?**

A video was published about the Table panel visualizations.
Added the YouTube link to the Table Panel Visualization page.

**Why do we need this feature?**

Add capacity to users to view how the steps are done.

**Who is this feature for?**

Everyone checking the docs about Table Panels.

**Which issue(s) does this PR fix?**:

N/A

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

srperf committed:
Update index.md added YouTube link to Table Panel

imatwawana reviewed (commented):
Thanks for this! I've left my thoughts about the placement of the video.

srperf commented on docs/sources/panels-visualizations/visualizations/table/index.md:
```diff
@@ -27,6 +27,9 @@ weight: 100
 
 # Table
 
+{{< youtube id="PCY7O8EJeJY" >}}
+
```
I placed it like the others, but I agree. Let's put it after 😄

imatwawana replied:
I should probably go back and look at the other ones that have been added again. I know one got merged before I had a chance to review it.

srperf replied:
@imatwawana 
Should we merge this before I add the descriptions mentioned before the video?

imatwawana replied:
It doesn't look like you've made the original change that I asked for to move the video to what is currently line 33/36. Do you want me to do that for you and then you can add the lead-in sentence?

srperf replied:
Oh! Yes please.
So sorry, I understood you'd do the change, I apologize for the confusion.
Please do and I will send the update after that :)

imatwawana committed:
Moved video and added placement note

imatwawana commented:
Sorry about the confusion! I've made the change and added a line to indicate where you should add an introductory sentence to the video.

srperf committed:
Update index.md adding video description

Added a description to the video. Feel free to edit if my syntax or writing needs improvement.

srperf commented:
All set @imatwawana 
I added the description of the video.
Hope it reads right, but if I did any Tarzan-like writing, please feel free to update as you see fit :)

imatwawana commented on docs/sources/panels-visualizations/visualizations/table/index.md:
```diff
@@ -31,6 +31,10 @@ Tables are very flexible, supporting multiple modes for time series and for tabl
 
 {{< figure src="/static/img/docs/tables/table_visualization.png" max-width="1200px" lightbox="true" caption="Table visualization" >}}
 
+The following video will give you a visual walkthrough over the configurations and settings you can set in a table panel. If you get stuck not knowing where is a setting in the documentation or want to see a configuration in action, check it out in the video below.
```
I took out the "if you get stuck in the documentation" part because I don't think we want to suggest that people are going to get stuck in the docs. Otherwise, I've just made some style fixes.

```suggestion
The following video provides a visual walkthrough of the configurations and settings you can set in a table visualization. If you want to see a configuration in action, check it out in the video:
```

imatwawana committed:
Wording and style edits

imatwawana commented on docs/sources/panels-visualizations/visualizations/table/index.md:
```diff
@@ -31,6 +31,10 @@ Tables are very flexible, supporting multiple modes for time series and for tabl
 
 {{< figure src="/static/img/docs/tables/table_visualization.png" max-width="1200px" lightbox="true" caption="Table visualization" >}}
 
+The following video provides a visual walkthrough of the options you can set in a table visualization. If you want to see a configuration in action, check it out in the video:
```
```suggestion
The following video provides a visual walkthrough of the options you can set in a table visualization. If you want to see a configuration in action, check out the video:
```

imatwawana committed:
Wording fix
